src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/menu/viewmodel/MainMenuViewModel.kt

```diff
@@ -32,7 +32,7 @@ class MainMenuViewModel : ViewModel() {
         refreshPlugins()
     }
 
-    fun importContainerDirectory(dir: File) {
+    fun importResourceContainer(dir: File) {
```

Comment:
the name dir should probably change if it's not exclusively a directory anymore (could be zip)

Reply:
done